packages/firestore/src/platform_node/grpc_connection.ts

```diff
@@ -135,26 +129,57 @@ export class GrpcConnection implements Connection {
         token
       };
     }
+
     return this.cachedStub.stub;
   }
 
-  private getRpc(rpcName: string, token: Token | null): any {
-    const stub = this.getStub(token);
-
+  private getRpcCallable<Req, Resp>(
+    rpcName: string
+  ): UnaryRpc<Req, Resp> | ReadableRpc<Req> | DuplexRpc {
     // RPC Methods have the first character lower-cased
     // (e.g. Listen => listen(), BatchGetDocuments => batchGetDocuments()).
     const rpcMethod = rpcName.charAt(0).toLowerCase() + rpcName.slice(1);
-    const rpc = stub[rpcMethod];
+    const rpc = this.cachedStub.stub[rpcMethod];
```

Comment:
Are you intentionally not calling ensureActiveStub() here? As-is this code isn't obviously correct, since I have to go figure out that some other code makes sure to call ensureActiveStub() before this code is called (I hope!).

Reply:
I should have called it here as well, but all callers of this method already called it. This was just a helper method of the helper methods, so it seemed safe.

Reply:
Safe yes, but obviously correct, no. :-)